src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -73,4 +78,21 @@ export class ContentTypesEditComponent extends BaseComponent {
         this.form.resetForm();
         this.readyToAddFields = true;
     }
+
+    private deleteContentType($event): void {
```

Comment:
This one should be public, because the ts lint should be crying about this: https://github.com/dotCMS/core-web/pull/385/files#diff-16ea9abe03b9527cca061fbb13631810R10